.github/workflows/code-coverage.yml

```diff
@@ -0,0 +1,185 @@
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - main
+      - master
```

Comment:
should we just remove master since we don't use that branch name anymore?

Reply:
Yeah we can. I'll remove it wherever it's being referenced.